c2corg_common/attributes.py

```diff
@@ -48,7 +49,9 @@
     'webcam',               # webcam
 
     'virtual',              # sur-WP virtuel
-    'misc'                  # divers
+    'misc',                 # divers
+
+    'slackline_spot',
```

Comment:
I would recommend to move ``slackline_spot`` up before ``misc`` because misc means "all other types", so it's better that it comes last.